File: cadwiki-nuget/cadwiki.DllReloader/ReloaderLog.vb
Imports cadwiki.NetUtils

Public Class ReloaderLog
    Inherits TextFileLog

    Public Overrides Property LogName As String
        Get
            Return MyBase.LogName
        End Get
        Set(ByVal value As String)
            MyBase.LogName = value
        End Set
    End Property

    Public Overrides Property LogDir As String
        Get
            Return MyBase.LogDir
        End Get
        Set(ByVal value As String)
            MyBase.LogDir = value
        End Set
    End Property

    Public Sub New()
        LogName = "ReloaderLog"
        CreateNewLogFile()
        DeleteOldLogFiles()
    End Sub
End Class

File: cadwiki-nuget/cadwiki.DllReloader/ReloaderLog.cs
using cadwiki.NetUtils;

namespace cadwiki.DllReloader
{

    public class ReloaderLog : TextFileLog
    {

        public override string LogName
        {
            get
            {
                return base.LogName;
            }
            set
            {
                base.LogName = value;
            }
        }

        public override string LogDir
        {
            get
            {
                return base.LogDir;
            }
            set
            {
                base.LogDir = value;
            }
        }

        public ReloaderLog()
        {
            LogName = "ReloaderLog";
            CreateNewLogFile();
            DeleteOldLogFiles();
        }
    }
}